modules/loader-utils/test/zip-utils/data-view-provider.spec.ts

```diff
@@ -1,7 +1,7 @@
 import test from 'tape-promise/tape';
-import {DATA_ARRAY} from '../data/test.zip.js';
-import {DataViewFileProvider} from '../../src/lib/parsers/parse-zip/data-view-file-provider';
-import {signature} from '../../src/lib/parsers/parse-zip/local-file-header';
+import {DATA_ARRAY} from '@loaders.gl/i3s/test/data/test.zip.js';
+import {DataViewFileProvider} from '../../src/parse-zip/data-view-file-provider';
```

Comment:
Not for this PR: Should this class (and colleagues) just be called `DataViewFile`? 
- The "provider" patter in JavaScript React is a kind of component that let's components communicate without passing props to each other. 
- Ignoring the fairly established provider pattern perhaps seems to me a provider could be some kind of factory that provides / lets you create objects.
- But probably not the objects (files) themselves?